libs/product/src/facades/composite-product/composite-product-facade.interface.ts

```diff
@@ -0,0 +1,31 @@
+import { Observable } from 'rxjs';
+import { Action } from '@ngrx/store';
+
+import { DaffStoreFacade } from '@daffodil/core';
+
+export interface DaffCompositeProductFacadeInterface extends DaffStoreFacade<Action> {
```

Comment:
Shouldn't the facade also expose the applied options?

Reply:
yea, whoops. Added